chapters/package-information.md

```diff
@@ -1,4 +1,25 @@
 # 7 Package information section
+In SPDX information is used to describe packages, representing the versioned components of an object that are part of a software distribution. Packages are an abstract concept that can be used to describe any object within a software distribution.
+
+A Package describes any unit of content that can be associated with a distribution of software. Typically, a Package is composed of one or more files. An SPDX document should provide details about the individual files comprising a software distribution, using a Package object to represent each versioned object within a distribution. Each package is identified with a classification type to aid with processing of the object described by a package element. Packages are used for objects that contain versioning information, where File is used to represent objects without versioning information, i.e. underconstruction.html.
+
+Any of the following non-limiting examples may be (but are not required to be) represented in SPDX as a Package:
+
+```
+a tarball, zip file or other archive
+a directory or sub-directory
+a separately distributed piece of software which another Package or File uses or depends upon (e.g., a Python package, a Go module, ...)
+a container image, and/or each image layer within a container image
+a collection of one or more sub-packages
+a Git repository snapshot from a particular point in time
+a open-source library, i.e. OpenSSL
+a javascript file
+a customer script file
+a index,html file
+a LICENSE file
+a CHANGE log 
+a properties file
+```
```

Comment:
@nishakm  this looks good. Will need to sync this material with section 5.2.2 which needs to be updated accordingly
https://github.com/spdx/spdx-spec/blob/development/v2.3/chapters/composition-of-an-SPDX-document.md
:
5.2.2 Package information section

If SPDX information is being used to describe packages, then one instance of the Package Information per package being described shall exist. It provides important meta information about the package as a whole. Packages are an abstract concept that can be used to refer to any distribution of software, typically consisting of one or more files and capable of containing sub-packages. Starting with SPDX 2.0, it is not necessary to have a package wrapping a set of files.

A Package refers to any unit of content that can be associated with a distribution of software. Typically, a Package is composed of one or more files. An SPDX document may, but is not required to, provide details about the individual files comprising a Package (see [Clause 8](https://spdx.github.io/spdx-spec/file-information/)).

Any of the following non-limiting examples may be (but are not required to be) represented in SPDX as a Package:

Reply:
Oh I see! I will update accordingly.